### ISSUE TITLE
Taking client "velocity" leads to wrong results

### ISSUE THREAD

**Issue by MWHunter** [MWHunter/ReaperAC#4]
Because the move method runs code after actually moving the player, which we don't when we just take the client's movements as the velocity

**Comment by MWHunter** [comment]
Fixed by calling the move method with the client velocity

This logic has been removed since it allowed clients to set the predicted clients, which seemed like a bad idea in the first place.  For example, a client could see their velocity to 100000 blocks per tick, and after the first violation, the anticheat would just kind of accept it.  Very bad idea and it's a good thing it was removed.  It might come back after more anti-exploit logic is in-place, as it does provide a precision boost to legitimate clients.

The best course of action is probably to add it in after all other logic is done, so I don't hide a problem with this precision boost.

**Comment by MWHunter** [comment]
Reopened as a reminder to myself to add this logic once the anticheat is done

**Comment by MWHunter** [comment]
Closing in favor of https://github.com/MWHunter/ReaperAC/issues/19 which is more specific to this issue
